<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainmenu">
     <widget class="QWidget" name="verticalWidget2" native="true">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>291</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Please select a mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdmin">
         <property name="text">
          <string>Admin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCustomer">
         <property name="text">
          <string>Customer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExit">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="adminloginpage">
     <widget class="QWidget" name="gridWidget2" native="true">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>431</width>
        <height>361</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <layout class="QGridLayout" name="login_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>10</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Welcome Admin</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="adminIDLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="AdminIDlabel">
         <property name="text">
          <string>Admin ID</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="adminpassLineEdit">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLineEdit" name="passwordLineEdit_2">
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="btnadminlogin">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="customerloginpage">
     <widget class="QWidget" name="gridWidget3" native="true">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>601</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <layout class="QGridLayout" name="login">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>10</number>
       </property>
       <item row="9" column="0">
        <widget class="QPushButton" name="btnloginacc">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="accpasslabel">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="accountIDlabel">
         <property name="text">
          <string>Account ID</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLineEdit" name="accpassLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="accountIDLineEdit"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Welcome, please enter your ID and password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="adminmainmenu">
     <widget class="QWidget" name="verticalWidget1" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>291</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>    Admin Main Menu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="adminbtn1">
         <property name="text">
          <string>Add customer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="adminbtn2">
         <property name="text">
          <string>Remove customer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="adminbtn3">
         <property name="text">
          <string>Back to main menu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="customermainmenu">
     <widget class="QWidget" name="verticalWidget3" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>291</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string> Customer Main Menu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="custbtn1">
         <property name="text">
          <string>Checking Account</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="custbtn2">
         <property name="text">
          <string>Savings Account</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="custbtn3">
         <property name="text">
          <string>Back to main menu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="addcustomerpage">
     <widget class="QWidget" name="gridWidget1" native="true">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>49</y>
        <width>541</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QLineEdit" name="pwLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Intial savings balance</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="checkingLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="idLineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>New customer ID</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="addCustomerBtn">
         <property name="text">
          <string>Add Customer</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Intial checking balance</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLineEdit" name="savingsLineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="removecustomerpage">
     <widget class="QFrame" name="verticalFrame">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>39</y>
        <width>211</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Customer ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="removeIDLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="removeCustomerBtn">
         <property name="text">
          <string>Remove Customer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="checkingaccountpage">
     <widget class="QFrame" name="gridFrame1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>291</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="0">
        <widget class="QPushButton" name="checkingBackBtn">
         <property name="text">
          <string>Back to menu</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="checkingWithdrawBtn">
         <property name="text">
          <string>Withdraw</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="checkingDepositBtn">
         <property name="text">
          <string>Deposit</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="checkingAmountLineEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="checkingTransferBtn">
         <property name="text">
          <string>Transfer to savings</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Enter your transaction amount</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="checkingBalanceLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="savingsaccountpage">
     <widget class="QFrame" name="gridFrame2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>301</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLineEdit" name="savingsAmountLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="savingsDepositBtn">
         <property name="text">
          <string>Deposit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="savingsBalanceLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Enter Deposit amount</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="savingsBackBtn">
         <property name="text">
          <string>Back to menu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuATM">
    <property name="title">
     <string>ATM</string>
    </property>
   </widget>
   <addaction name="menuATM"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
